OpenRA.Game/Graphics/Animation.cs

```diff
@@ -22,7 +22,7 @@ public class Animation
 		public string Name { get; private set; }
 		public bool IsDecoration { get; set; }
 
-		readonly SequenceProvider sequenceProvider;
+		readonly SequenceProvider sequences;
```

Comment:
I personally dislike the `sequenceProvider` -> `sequences` rename when it's for a `SequenceProvider` instance.
I get that it may have been for consistency because we currently use both names for it, but I do not enjoy the direction this took.

Reply:
I see - preparation for later `SequenceProvider` rename/rework.